Vic2ToHoI4/Data_Files/configurables/cultureGroupToGraphics.txt

```diff
@@ -33,6 +33,10 @@ germanic = {
 		neutrality = { "gfx/interface/ideas/idea_albert_kesselring.dds" }
 		radical = { "gfx/interface/ideas/idea_hjalmar_schacht.dds" } 
 	}
+	male_monarch_portraits = {
```

Comment:
Not related to this PR but you're mixing filenames with and without extensions. Better to unify the style if possible.

Reply:
ack